src/components/Me/Navigation/Navbar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './Navbar.css';
+import IconHome from './IconHome';
+import IconMentors from './IconMentors';
+
+const Navbar = () => {
+  return (
+    <div>
+      <div className="menu-tablet">
+        <div className="logo-left">
+          <img
+            class="sidenav-logo"
+            src={`${process.env.PUBLIC_URL}/codingcoach-logo-192.png`}
+            alt=""
+          />
+        </div>
+        <div className="house side-icon">
+          <a href="/" className="icon">
+            <div>
+              <IconHome />
+              <span className="desc">Home</span>
+            </div>
+          </a>
+        </div>
+        <div className="mentors side-icon">
+          <a href="#" className="icon">
+            <div>
+              <IconMentors />
+              <span className="desc">Mentors</span>
+            </div>
+          </a>
```

Comment:
Please use `<Link />` to use `react-router` instead of regular link.
`import {Link} from 'react-router-dom';`

Also, this link should link to `/` - which is the mentors list

```suggestion
          <Link to="/" className="icon">
            <div>
              <IconMentors />
              <span className="desc">Mentors</span>
            </div>
          </Link>
```